wallet/src/de/schildbach/wallet/ui/WalletActivity.java

```diff
@@ -331,10 +328,26 @@ public void onActivityResult(final int requestCode, final int resultCode, final
     }
 
     private void handleString(String input, final int errorDialogTitleResId, final int cannotClassifyCustomMessageResId) {
```

Comment:
I also wanted to move this method into the view model, but this would create problems with some of the tests. Leaving it here for now.